src/gateway/requests-from-cache.js

```diff
@@ -40,7 +40,7 @@ export const asUncachedRequest = (
      * We need to ensure that what we return has the `abort` method still so
      * that we can let things like JSDOM call abort on promises.
      */
-    const superagentRequest = request.buffer(options.buffer);
+    const superagentRequest = request.buffer(true);
```

Comment:
We could pass nothing here; that is also equivalent to `true`, but I'd rather be explicit.